README.rst

```diff
@@ -18,15 +18,34 @@ Installation
 Usage
 =====
 
-::
+Running a match:
+
+.. code-block:: python
+
+   >>> import axelrod_fortran as axlf
+   >>> import axelrod as axl
+   >>> p1 = axlf.Player('k31r')
+   >>> p2 = axlf.Player('k33r')
+   >>> match = axl.Match((p1, p2), turns=5)
+   >>> match.play()
+   [(C, C), (C, C), (C, D), (C, D), (C, C)]
+
+Running an instance of Axelrod's second tournament:
+
+.. code-block:: python
+
+   >>> import axelrod_fortran as axlf
+   >>> import axelrod as axl
+   >>> players = [axlf.Player(name) for name in axlf.characteristics.keys()
+   >>>            if axlf.characteristics[name]["original_rank"] is not None]
```

Comment:
> all-strategies should be the list of strategies that participated in the second tournament.

`all_strategies` in `axelrod` points at the entire list of strategies (including the cheaters etc).

I think that would be nice to keep here too (so there still is a list of **all** the strategies) but then we can create a second list that is @marcharper's comprehension that would be something like `axlf.tournament_strategies`?